### DISCUSSION TITLE
Problem with commiting files to LFS via web UI

### DISCUSSION THREAD

**Discussion by donald-berning** [community/community#12705] [Other Feature Feedback, Questions, & Ideas]
See [Problem with commiting files to LFS via web UI](https://github.community/t/problem-with-commiting-files-to-lfs-via-web-ui/119605)
LFS "upload image file via web UI on github.com it is just added in repo (not in git LFS)."
"pushing files to LFS via our web UI is not supported."
"The web UI lacks these hooks, and can’t consult the .gitattributes file to see what types of files it should deal with. (Similarly, it doesn’t check the .gitignore file so you can create and commit files from the ignore list directly in the UI if you so choose.)"

The lack of LFS upload support in the web UI causes considerable grief when it becomes necessary to handle a local clone of the repository (files that should have been pointers).  The web UI is able to read LFS files (thank goodness).  

Please _fix_ the web UI to also support writing LFS files (read and respect the .gitattributes).

Web UI ignoring the .gitignore is also a bug that needs to be fixed.

**Comment by sidharthramesh** [comment]
+1

**Comment by Bulalu** [comment]
any updates to this??

**Comment by lsb** [comment]
I'd love to have the opportunity to pay to use more LFS storage from the web ui!
